cf_xarray/accessor.py

```diff
@@ -2510,6 +2646,20 @@ def formula_terms(self) -> dict[str, str]:
             terms[key] = value
         return terms
 
+    @property
+    def grid_mapping(self) -> DataArray | None:
```

Comment:
Could also add a `grid_mapping_name` property to get the string if that seems useful to  you.

Reply:
Yes, i think, that makes sense here!